Dockerfile.integration-test

```
@@ -0,0 +1,14 @@
+FROM bats/bats:latest
+
+ARG KUBECTL_VERSION=v1.29.2
+
+# Add packages
+RUN apk --no-cache add \
+    curl \
+    git \
+    bash
```

I would run an upgrade on all installed packages to fix potential CVEs in advance
```suggestion
RUN apk upgrade --no-cache --no-progress \
&& apk --no-cache --no progress add curl git bash \
&& apk del --no-cache --no-progress apk-tools alpine-keys
```

---

I have fixed in https://github.com/cloudoperators/greenhouse-extensions/pull/237/commits/924cf43397727fe509b9c81e936192123c6d484f commit.